salt/api-gateway/config/etc-kong-kong.conf

```diff
@@ -54,57 +54,53 @@ anonymous_reports = off          # Send anonymous usage data such as error
 
 
 # modified
-proxy_listen = 0.0.0.0:80     # Address and port on which Kong will accept
+#proxy_listen = 0.0.0.0:80       # Address and port on which Kong will accept
```

Comment:
The master branch does not work on Vagrant due to errors relative with the dummy `fullchain.pem` and related files. Switching to the branch.